### PR TITLE
fix: Remove erroneous style keys

### PR DESCRIPTION
**Pull request by kazo0**

### CODE REVIEW

File: src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml [hunk 1]
```diff
@@ -142,7 +142,7 @@
 				Value="Compact" />
 		<Setter Property="ExitDisplayModeOnAccessKeyInvoked"
 				Value="False" />
-		<Setter Property="CornerRadius"
+		<not_win:Setter Property="CornerRadius"
```
Comment (agneszitte-nventive) [inline src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml:145]: trying to understand why we need "not_win" here, can you give me a little more details please ?

**Review by agneszitte-nventive — APPROVED** [review]

File: src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml (hunk 1 above)
Comment (agneszitte-nventive) [inline src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml:145]: This mean we will note have CornerRadius for WinUI @kazo0 ?

Comment (agneszitte-nventive) [inline src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml:145]: This mean we will note have CornerRadius for WinUI @kazo0 ?

**Review by agneszitte-nventive — COMMENTED** [review]
This mean we will note have CornerRadius for WinUI @kazo0 ?

File: src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml (hunk 1 above)
Comment (kazo0) [reply src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml:145]: Honestly I don't know right now. It seems like you can't reference the ControlCornerRadius resource when running a WinUI app

Comment (kazo0) [reply src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml:145]: I've apparently done this before and I have no memory of this: https://github.com/unoplatform/Uno.Themes/pull/586

Not sure if maybe @jeromelaban would know why we needed to put either `not_win:` or `contract7NotPresent:`

**Pull request merged**